docs/publish/README.md

```diff
@@ -1,197 +1,157 @@
 # Publish
 
+## Publishing a Concept
 
-## Publish a Concept
-
-To publish a **Concept**, go to the **Concept** page with the navigation menu.
+To publish a **Concept**, navigate to the **Concepts** page from the project navigation menu.
 
 ![Concept Menu](../img/getting-started/menu_concept.png)
 
-To upload a concept, click the **Add a new reference to concepts** button.
-
-You can upload one or several concepts at the same time.
+To upload a concept, click the **Add a new reference to concepts** button. You can upload one or several concepts simultaneously.
 
 ![Concept empty page](../img/getting-started/concept_empty_prod.png)
 
-Once you upload your previews, the concept page will look like this.
+After the upload is complete, previews will be generated and visible from your concepts page.
 
 ![Concept filled page](../img/getting-started/concept_filled_prod.png)
 
+Click on the thumbnail to see an enlarged preview of your concept, or click on the status to open the **Comment Panel** on the right.
 
-You can interact with the concept in two ways: click on the picture to see an enlarged view.
-The second is to click on the status part to open the **Comment Panel** on the right.
+With the comment panel open, you have two options: 
 
-You have two options on the comment panel: link a concept with an existing asset or delete the concept.
-You can also comment and change the status of the asset.
+1) You can link a concept with an existing asset / delete and existing link. 
+2) You can comment and change the status of the asset.
+<!-- do you mean status of the concet, or asset? -->
 
-The idea is to have one Version per **Concept**. If it's not validated, you need to upload a new concept, not to have multiple versions of the same concept.
-
-One concept is one task.
+It is good practice to only have one version per **Concept**. If the concept is not approved and requires additional changes, then it's better to version-up that concept.
 
+One concept equals one task.
+<!-- need to double check this -->
```

Comment:
It's not very clear. I think we should not talk about a task there. People comment and review concepts directly. The underlying data model is a task but from a user perspective, it's a concept.

Reply:
Thanks, I have removed this line to avoid confusion